mlte/property/property.py

```diff
@@ -35,91 +36,60 @@ def __init__(self, name: str, description: str, rationale: str):
         """
         self.name: str = name
         """The name of the property."""
-        self.description = description
+
+        self.description: str = description
         """The description of the property."""
-        self.rationale = rationale
+
+        self.rationale: str = rationale
         """The rationale for using the property."""
 
-    def _to_json(self) -> Dict[str, str]:
+    def to_model(self) -> PropertyModel:
         """
-        Save a Property instance to a JSON document.
+        Return a Property as a model.
 
-        :return: The document
-        :rtype: Dict[str, str]
+        :return: The property as its model.
+        :rtype: PropertyModel
```

Comment:
Just for your knowledge: I switched our documentation generator from `sphinx` to `mkdocs`. The API reference generator for `mkdocs` can infer types for docstrings from Python type annotations, so we no longer need to annotate the parameter types and return types explicitly. This is not a big deal so I won't comment on it any further; eventually we'll just go through and remove the explicit annotations.